five-principles-that-guide-tidb-and-pingcap.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Five Principles that Guide TiDB and Our Company (Part I)
+author: ['Max Liu']
+date: 2021-01-21
+summary: PingCAP's CEO talks about the philosophy of TiDB's evolution and how PingCAP built TiDB in the past five years.
+tags: ['Architecture']
+categories: ['Company']
+image: /images/blog/tidb-philosophy-five-principles-that-guide-tidb-and-our-company.jpg
+---
+
+![Five Principles that Guide TiDB and Our Company](media/tidb-philosophy-five-principles-that-guide-tidb-and-our-company.jpg)
```

Comment:
```suggestion
![Five Principles that Guide TiDB and PingCAP](media/tidb-philosophy-five-principles-that-guide-tidb-and-our-company.jpg)
```

Reply:
Next time, you could use tidb-philosophy-five-principles-that-guide-tidb-and-pingcap.jpg as the image name.

Reply:
Thanks. Fixed~